renderer/components/TrackTimeModal/TrackTimeModal.tsx

```diff
@@ -473,7 +473,7 @@ export default function TrackTimeModal({
       <Dialog
         as="div"
         className="fixed inset-0 z-10 overflow-y-auto"
-        onClose={close}
+        onClose={() => null}
```

Comment:
Is it possible to remove onClose as props and don't provide anything to the Dialog component?

Reply:
no, this prop is required